bears/ruby/RubySmellBear.py

```diff
@@ -162,7 +162,7 @@ def generate_config(self,
             "UncommunicativeModuleName": bad_module_name,
             "UncommunicativeParameterName": bad_param_name,
             "UncommunicativeVariableName": bad_var_name,
-            "UnusedParameters": check_unused_variables,
+            "UnusedParameters": not allow_unused_variables,
             "UnusedPrivateMethod": check_unused_private_methods,
```

Comment:
^^ Ah nvm. You have already made the suggested changes in abdf20376c5f41e64457110d99e01f274d1adb5f .
Btw just fix the commit message :+1: